plasmapy/particles/particle_class.py

```diff
@@ -90,6 +91,22 @@
 """
 
 
+class SetFormatter:
```

Comment:
```suggestion
class _SetFormatter:
```
...to mark it as a private class.